bulk-targeted-sponsorship/README.md

```diff
@@ -0,0 +1,381 @@
+
+As part of experimenting through different approaches, Microsoft trialed a data-driven
+approach in June of 2024 that leveraged the availability of good metadata on 
+packages and their source repositories combined with metadata on the usage of 
+dependencies across our many tens of thousands internal repositories.
+Our hypothesis was this targeted bulk approach would enable sponsorships to target
+open source projects that were less well known while also being
+highly impactful as judged by public download stats, downstream dependent package counts,
+and Microsoft internal usage. The final list appears to support these initial hypotheses
+as there are many packages never previously submitted for FOSSFUND.
+Additionally, while past FOSS FUND awardees have almost always been organizations,
+this round saw many individuals who develop an impactful open source project
+get sponsored. 
+
+This document attempts to summarize our approach in very broad terms and describe
+both what we did and more basic approaches others might be able to take who lack the
+benefits of our internal inventory of engineering systems.
+
+## Dependency Inventory
+### _To target dependencies at scale, you need a full listing of which ones you use_
+
+Knowing your dependency usage counts is a necessary step for targeted sponsorship. 
+
+### Advanced: inventory
+Microsoft has a centralized team that is responsible for collecting
+engineering system metadata. They provide various tables in 
+[Azure Data Explorer](https://azure.microsoft.com/en-us/products/data-explorer/)
+that the Open Source Programs Office has leveraged to get repository counts 
+per unique dependency. This capability allows us to not just count dependency usage,
+but also analyze usage by public vs. internal repositories as well as other dimensions.
+
+
+### Basic: inventory
+
+Even if you do not have the benefit of a centralized team already collecting this data,
+there are options for getting a mostly complete view without too much effort. 
+Microsoft saw a high degree of overlap between the dependencies of our public-facing
+repositories in GitHub and our internal repositories. 
+If we had only used data from public repositories via GitHub's SBOM export API,
+our final sampling for sponsorship would not have been too different.
+
+#### GitHub SBOM 
+
+As seen in the GitHub 
+[blog post announcing Self-service SBOMs](https://github.blog/2023-03-28-introducing-self-service-sboms/), 
+you can manually go to the insights tab of any public-facing GitHub repository and download a
+SBOM (Software Bill of Materials) JSON file that contains information on all the dependencies of
+that repository. This is available for every public GitHub repository whether you own it or not.
+
+The example SBOM JSON shown below comes from clicking the "Export SBOM" button
+on this page: https://github.com/microsoft/opensource.microsoft.com/network/dependencies
```

Comment:
this should also be a link in markdown no?

Reply:
Could but spelled out so people could see the URL structure and pop in their own repos. Just clicking around it is a bit hard to find.